docs/api/plotting/index.rst

```diff
@@ -0,0 +1,56 @@
+Plotting
+========
+
+* Intuitive plotting routines with ``matplotlib`` similar syntax (see :ref:`plotting_ref`).
+* Plotting tools built for interactivity (see :ref:`widgets`).
+
+.. toctree::
+   :maxdepth: 2
+
+   plotting
+   qt_plotting
+
+
+Plotting API Reference
+----------------------
+Plotting module API reference.  These plotting modules are the basis for
+all plotting functionality in PyVista.
+
+.. currentmodule:: pyvista
+
+.. autosummary::
+   :toctree: _autosummary
+   :template: custom-class-template.rst
+
+   BasePlotter
+   Plotter
+   Renderer
+
+
+Widget API
+----------
+The :class:`pyvista.BasePlotter` class inherits all of the widget methods in
+:class:`pyvista.WidgetHelper`, so all of the following methods
+are available from any PyVista plotter.
+
+.. currentmodule:: pyvista
+
+.. autosummary::
+   :toctree: _autosummary
+   :template: custom-class-template.rst
+
+   WidgetHelper
+
+
+Convenience Functions
+~~~~~~~~~~~~~~~~~~~~~
+These functions provide a simplified interface to the various plotting
+routines in PyVista.
+
+.. autosummary::
+   :toctree: _autosummary
+
+   plot
+   plot_arrows
+   set_plot_theme
+   create_axes_orientation_box
```

Comment:
Would it be possible to put these in a sublist of their own? Perhaps that would make the left sidebar feel less cluttered. They had a separate link of their own originally:
![Screenshot 2021-06-17 at 17-05-12 Plotting — PyVista 0 31 dev0 documentation](https://user-images.githubusercontent.com/17914410/122423430-34054b80-cf8e-11eb-8b23-b45897883075.png)

Reply:
Agreed.  I found a way to nest it.